simbad/rotsearch/amore_search.py

```diff
@@ -560,3 +243,89 @@ def summarize(self, csv_file):
         ]
         summarize_result(self.search_results,
                          csv_file=csv_file, columns=columns)
+
+    def _generate_hklpck0(self):
+        logger.info("Preparing files for AMORE rotation function")
+        f, sigf, _, _, _, _, _ = simbad.util.mtz_util.get_labels(self.mtz)
+        stdin = self.sortfun_stdin_template.format(f=f, sigf=sigf)
```

Comment:
Out of curiosity, where is the template specified?

Reply:
~20 lines down

Reply:
cheers, couldn't see it for the life of me just then.

Reply:
I moved all `property` attributes to the bottom for clarity of the functions near the top.